frontend/src/metabase/query_builder/components/notebook/NotebookStep.jsx

```diff
@@ -124,6 +124,7 @@ export default class NotebookStep extends React.Component {
             mt={isLastStep ? 2 : null}
             large={largeActionButtons}
             {...(STEP_UI[action.type] || {})}
+            key={'actionbutton_' + STEP_UI[action.type].title}
```

Comment:
```suggestion
            key={`actionButton_${STEP_UI[action.type].title`}
```
This is a slightly nicer way to do string concatenation and `actionButton` reads a little more clearly to me.

Reply:
Ah, right. I had some default map (` || {}`) in there and thought that that would look a bit messy in a template string. Forgot to change back to template string when I removed the default. Thanks! :+1: